hibernate-core/src/main/java/org/hibernate/resource/beans/container/internal/JpaCompliantLifecycleStrategy.java

```diff
@@ -125,7 +125,7 @@ public void initialize() {
 				throw e;
 			}
 			catch (Exception e) {
-				log.debugf( "Error resolving CDI bean [%s] - using fallback" );
+				log.debugf( "Error resolving CDI bean [%s] - using fallback", beanType.getName() );
```

Comment:
It is still a common bug to miss format parameter. I tried the Checker Framework (https://checkerframework.org/) but it crashed when I used it on hibernate-orm, :(.